<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EtemenankiClass</class>
 <widget class="QMainWindow" name="EtemenankiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Etemenanki</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="begin_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>400</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin</string>
    </property>
   </widget>
   <widget class="QListView" name="regexList">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="terminal_output">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Etemenanki.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>EtemenankiClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>begin_button</sender>
   <signal>released()</signal>
   <receiver>EtemenankiClass</receiver>
   <slot>handleButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>770</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleButton()</slot>
 </slots>
</ui>
